man/ocamlc.m

```diff
@@ -493,6 +493,13 @@ object code files (.cmo), libraries (.cma), and C libraries specified
 cmo file, and also sets the module name to the file name up to the
 first dot.
 .TP
+.B \-opaque
+When producing a .cmi file, this option mark the module as opaque:
```

Comment:
Here and below, there's a text of the form "-opaque means opaque", which doesn't say anything :)

A few additional thoughts:
- It's probably worth explicitly pointing out one main use of this option, which is improved compilation speed.
- We should also be careful with "dependencies" versus "dependents" (dependencies of module X being the things needed to compile module X, and the dependents being the things that need X in order to compile).
- I haven't checked the code, but please see what happens if the .cmx file exists and the .cmi says that it's opaque, if you haven't already.
- I tend to think we should be a bit careful with "module" versus "compilation unit".

A first attempt at improving this paragraph (wording may need adjusting depending on 3rd bullet point above):

"Mark the compilation unit so that other modules using it will only have access to its interface and not any details of its implementation.  This can improve speed of compilation, both from clean and incrementally, at the expense of performance of the generated code."